src/components/SomeGraph.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+  import { DataGraphic } from "@graph-paper/datagraphic";
+  import { Line } from "@graph-paper/elements";
+  import { LeftAxis, BottomAxis } from "@graph-paper/guides";
+
+  export let points = 100;
+  export let size = 1;
+  export let curve = "curveMonotoneX";
+  export let dashArray = "1,0";
```

Comment:
The `size`, `curve`, and `dashArray` params have defaults in the `Line` component which match these so these aren't needed at all, so below this could be simply `<Line {data} />`.